src/Sulu/Bundle/AdminBundle/Resources/public/js/components/login/main.js

```diff
@@ -62,98 +58,118 @@ define([], function() {
-                            '      <input id="user" class="form-element input-large husky-validate" type="text" placeholder="<%= emailUser %>" tabindex="-1"/>',
-                            '    </div>',
-                            '    <span class="error-message"></span>',
-                            '  </div>',
-                            '  <div class="grid-row">',
-                            '      <div id="'+ constants.resetMailButtonId +'" class="btn action large fit"><%= reset %></div>',
-                            '  </div>',
-                            '</div>',
-                            '<div class="'+ constants.resetMailMessageBoxClass +'">',
-                            '<span class="message"><%= sentMessage %> <span class="to-mail"></span></span>',
-                            '   <div id="'+ constants.resendButtonId +'" class="btn action large fit"><%= resend %></div>',
-                            '</div>',
-                            '<div class="bottom-container small-font">',
-                            '  <span class="'+ constants.websiteSwitchClass +'"><%= backWebsiteMessage %></span>',
-                            '  <span class="'+ constants.loginSwitchClass +'"><%= backLoginMessage %></span>',
-                            '</div>'].join(''),
-                resetFrame: ['<div class="'+ constants.resetMailBoxClass +'">',
-                             '  <div class="inputs">',
-                             '    <div class="grid-row">',
-                             '      <label for="password1"><%= password1Label %></label>',
-                             '      <input id="password1" class="form-element input-large husky-validate" type="password" placeholder="<%= password %>"/>',
-                             '    </div>',
-                             '    <div class="grid-row small">',
-                             '      <label for="password2"><%= password2Label %></label>',
-                             '      <input id="password2" class="form-element input-large husky-validate" type="password" placeholder="<%= password %>"/>',
-                             '    </div>',
-                             '  </div>',
-                             '  <div class="grid-row">',
-                             '      <div id="'+ constants.resetButtonId +'" class="btn action large fit"><%= login %></div>',
-                             '  </div>',
-                             '</div>',
-                             '<div class="bottom-container small-font">',
-                             '  <span class="'+ constants.websiteSwitchClass +'"><%= backWebsiteMessage %></span>',
-                             '  <span class="'+ constants.loginRouteClass +'"><%= loginRouteMessage %></span>',
-                             '</div>'].join('')
+            mediaContainer: ['<div class="' + [constants.mediaContainerClass, constants.mediaLoadingClass].join(" ") + '">',
```

Comment:
@nnatter make sure to only make class-names and ids constant that need to be targeted via js later on. I'd prefer to classes and ids which are only needed for styling to be directly coded into the template as it makes the code more readable